src/NexusMods.App.UI/Pages/ModLibrary/FileOrigins/FileOriginEntry/FileOriginEntryViewModel.cs

```diff
@@ -83,5 +88,14 @@ public class FileOriginEntryViewModel : AViewModel<IFileOriginEntryViewModel>, I
             .Merge(interval.Select(_ => LastInstalledDate))
             .Select(date => date.Equals(DateTime.MinValue) ? "-" : date.Humanize())
             .ToProperty(this, vm => vm.DisplayLastInstalledDate, scheduler: RxApp.MainThreadScheduler);
+
+        ViewModCommand = viewModCommand;
+    }
+    
+    public async Task AddToLoadout(CancellationToken token) => await AddUsingInstallerToLoadout(null, token);
+
+    public async Task AddUsingInstallerToLoadout(IModInstaller? installer, CancellationToken token)
+    {
+        await _archiveInstaller.AddMods(_loadoutId, FileOrigin, installer, token);
```

Comment:
This stuff can be put in FileOriginsPageViewModel instead of putting it inside the entry VM.
Parent is the one doing multi selection operation, and it shouldn't invoke the commands of the children to do the operation, it should do it directly. So it should be in charge of all the add mod operations. 


In the future we might handle multi installs in a more sensible manner than calling AddMod() multiple times.